cmake.deps/cmake/BuildWasmtime.cmake

```diff
@@ -0,0 +1,10 @@
+get_externalproject_options(wasmtime ${DEPS_IGNORE_SHA})
+ExternalProject_Add(wasmtime
+  DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}/wasmtime
+  CONFIGURE_COMMAND ""
+  BUILD_COMMAND cargo build --release -p wasmtime-c-api
```

Comment:
TODO(dundargoc): this will always use release which isn't great. Could use a genex here to fix it. Alternatively we could fix the cmake file in wasmtime and use that instead.

Reply:
Nope, cmake is broken, I can't for the life of me get genex to work here. Always default to `--release` for the time being.